packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3337,6 +3373,13 @@ class SemanticsConfiguration {
     _isSemanticBoundary = value;
   }
 
+  /// Whether to block user interactions for the semantics subtree.
+  ///
+  /// Setting this true prevents user from activating pointer related
+  /// [SemanticsAction]s, such as [SemanticsAction.tap] or
+  /// [SemanticsAction.longPress].
+  bool isBlockingUserInteractions = false;
```

Comment:
Random thought: Is there ever a use case for only blocking certain actions? I wonder if this should just take a list of blocked actions (that we internally translate to a bitMask right away) and provide a const value containing all block-able actions that the use cases in this PR would use. Would be a little more flexible, but not sure how useful that is...

Reply:
I haven't found use cases other than blocking pointer related action. We can add a new api to block a list instead of just hardcoded actions in the future if we have more use cases, and then we can reimplment this boolean to use that API under the hood.

I was suspecting this may be an one-off case, that is why I go with hardcoded actions.